tests/orbits/keplerian_test.py

```diff
@@ -59,6 +59,17 @@ def test_keplerian_central_density():
     )
 
 
+def test_keplerian_central_from_orbit():
+    sun = keplerian.Central.from_orbital_properties(
```

Comment:
I think it should be enough here to just remove the `keplerian.`:

```suggestion
    sun = Central.from_orbital_properties(
```

since `Central` is imported up top.